e2e/test/scenarios/dashboard/reproductions/13736-dashboard-should-work-if-some-cards-are-broken.cy.spec.js

```diff
@@ -0,0 +1,48 @@
+import {
+  getDashboardCards,
+  restore,
+  updateDashboardCards,
+  visitDashboard,
+} from "e2e/support/helpers";
+
+describe("issue 13736", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should work even if some cards are broken (metabase#13736)", () => {
+    cy.intercept("POST", "/api/dashboard/*/dashcard/*/card/1/query", {
+      statusCode: 500,
+      body: {
+        cause: "some error",
+        data: {},
+        message: "some error",
+      },
+    });
+
+    cy.createDashboard({ name: "13736 Dashboard" }).then(
+      ({ body: { id: dashboardId } }) => {
+        updateDashboardCards({
+          dashboard_id: dashboardId,
+          cards: [
+            {
+              card_id: 1,
```

Comment:
Hardcoding question ids is problematic given Audit v2 changes. Can we avoid doing that? E.g. we can create questions if needed

Reply:
Gotcha. I thought this would be the simplest thing to do. But creating new questions would be more deterministic.

Reply:
Fixed in 14d8020e74369cb2508ea44cd4e9a90a42ddc2eb